Library/OcMainLib/OpenCoreMisc.c

```diff
@@ -259,37 +258,26 @@ OcToolDescribeEntry (
 {
   EFI_STATUS          Status;
   CHAR16              DescPath[OC_STORAGE_SAFE_PATH_MAX];
-  OC_STORAGE_CONTEXT  *Storage;
   BOOLEAN             HasIcon;
   BOOLEAN             HasLabel;
 
-  Storage  = (OC_STORAGE_CONTEXT *) Context;
   HasIcon  = FALSE;
   HasLabel = FALSE;
 
   if (LabelData != NULL && LabelDataSize != NULL) {
     *LabelData     = NULL;
     *LabelDataSize = 0;
 
-    if (ChosenEntry->Type == OC_BOOT_RESET_NVRAM) {
-      Status = OcUnicodeSafeSPrint (
-        DescPath,
-        sizeof (DescPath),
-        OPEN_CORE_LABEL_PATH "ResetNVRAM.%a",
-        LabelScale == 2 ? "l2x" : "lbl"
-        );
-    } else {
```

Comment:
To be clear, I'm really not sure this removed code made sense in the first place - it is loading the ResetNVRAM icon from the OpenCore LABELS path. But that is already in the pre-loaded system labels.